pandas/_libs/src/numpy.pxd

```diff
@@ -598,8 +603,10 @@ cdef extern from "numpy/arrayobject.h":
     object PyArray_Dumps (object, int)
     int PyArray_ValidType (int)
     void PyArray_UpdateFlags (ndarray, int)
-    object PyArray_New (type, int, npy_intp *, int, npy_intp *, void *, int, int, object)
-    #object PyArray_NewFromDescr (type, dtype, int, npy_intp *, npy_intp *, void *, int, object)
+    object PyArray_New (type, int, npy_intp *, int, npy_intp *,
+                        void *, int, int, object)
+    #object PyArray_NewFromDescr \
```

Comment:
you can't use ``\`` here, this is kind of like a c-header file. you can just remove this line as its not used anyhow (its commented out)